content/release-10-7-0/platform-10-7-0-bundle/10-7-0-24.md

```diff
@@ -208,7 +208,7 @@ MTM-32730</td>
 <tr>
 <td>
 REST API</td>
-<td > Deletion of audit logs is no longer permitted. All DELETE requests to the audit API should return the error “405 Method not allowed”. See also <a href="/release-10-7-0/migration-10-7-0" class="no-ajaxy">Release 10.7.0 > Migration notes</a>.</td>
+<td > Deletion of audit logs is no longer permitted. All DELETE requests to the audit API should return the error “405 Method not allowed”.</td>

```

Comment:
The link should stay, but point to announcements-10-7-0 (Release 10.7.0 > Important announcements)

Reply:
Fixed